exe/genv-envs

```diff
@@ -88,13 +119,31 @@ def do_activate(envs: Envs, eid: str, uid: int, username: Optional[str], pid: Op
 
         env["kernel_ids"].append(kernel_id)
 
+
 def do_find(envs: Envs, pid: Optional[int], kernel_id: Optional[str]) -> None:
+    """
+    Query genv environments to find which one is related to the given process/kernel.
+    :param envs: dictionary describing all existing environments.
+    :param pid: process id to query for (optional)
+    :param kernel_id: jupyter kernel id to query for (optional).
+    :return: Return appropriate environments id
```

Comment:
it actually does not _return_ it (it returns `None`) but _prints_ it 👌

Reply:
Fixed